doc/dev/developer_guide/dash-devel.rst

```diff
@@ -439,7 +439,7 @@ Note:
 Start E2E tests by running::
 
   $ cd <your/ceph/repo/dir>
-  $ sudo chown -R $(id -un) src/pybind/mgr/dashboard/frontend/dist src/pybind/mgr/dashboard/frontend/node_modules
+  $ sudo chown -R $(id -un) build/src/pybind/mgr/dashboard/frontend/dist src/pybind/mgr/dashboard/frontend/node_modules
```

Comment:
```suggestion
  $ sudo chown -R $(id -un) src/pybind/mgr/dashboard/frontend/dist src/pybind/mgr/dashboard/frontend/node_modules
```

@tchaikov This is not needed, it works as it is without the need of any build folder.

Reply:
but `dist` is now created under `build/src/pybind/mgr/dashboard/frontend/` now . do you mean the E2E tests are always launched in the source tree uncompressed from a dist tarball?

Reply:
Yes, KCLI E2E tests are run like this:
```
kcli create plan -f ./src/pybind/mgr/dashboard/ci/cephadm/ceph_cluster.yml \
    -P ceph_dev_folder=${CEPH_DEV_FOLDER} \
    ${EXTRA_PARAMS} ceph
```
Where ceph_dev_folder is the PR source code tested in jenkins job.

In turn, ceph_dev_folder is used for setting `--shared_ceph_folder` in cephadm:
https://github.com/ceph/ceph/blob/master/src/cephadm/cephadm#L8004

Is this refactoring affecting the developer mode of cephadm?

Reply:
@alfonsomthd thank you. i see. since `pybind/mgr/dashboard/ci/cephadm/run-cephadm-e2e-tests.sh` runs `npm run build` right under `src/pybind/mgr/dashboard/frontend` instead of using `cmake`, i think this changeset does not change the place where `dist` is created if `run-cephadm-e2e-tests.sh` is part of the workflow.

reverted and repushed.

Reply:
I think this does not impact the cephadm developer mode.